vng_api_common/audittrails/models.py

```diff
@@ -66,6 +66,16 @@ class AuditTrail(models.Model):
         encoder=DjangoJSONEncoder,
         help_text=_("Volledige JSON body van het object na de actie")
     )
+
+    gebruikers_id = models.CharField(
```

Comment:
probably needs a `blank=True` as well - very few people will actually implement this immediately